src/app/services/logger/logger.module.ts

```diff
@@ -0,0 +1,19 @@
+import {NgModule, CUSTOM_ELEMENTS_SCHEMA} from '@angular/core';
```

Comment:
this needs to be removed and the ExuiLoggerComponent needs to be added to apps shared.module

Reply:
Done. Thanks